### DISCUSSION TITLE
What's new and which schematic to use?

### DISCUSSION THREAD

**Discussion by delchrys** [emsesp/EMS-ESP32#539] [Q&A]
Hi I've been working on this project before but with a esp8266 at that time.
Like to pick it up again and aybe get it working properly for me at this time.
Are there any new schematics for the hardware to use? For the ems message decode part....
At the moment I'm in possession of several mictoco trollers so I can uzrarduino/esp8266 or esp32, I have them all.
Just want to have latest version of schematics and software.
Kind regards,
Bert

**Comment by MichaelDvP** [comment]
I think the interface cicuit is not changed and ca be found [here](https://emsesp.github.io/docs/#/Hardware), [here](https://github.com/bbqkees/Nefit-Buderus-EMS-bus-Arduino-Domoticz/tree/master/PCB-files/V0.9), and [here](https://github.com/rocksolidsr/esp-ems-board).
Latest software is only for esp32 in the dev branch on this page, the v2 version for 8266 is outdated and not recommended, [see here](https://github.com/emsesp/EMS-ESP).

You can read all this in the [documentation](https://emsesp.github.io/docs/#/Home).
